test/looker/test_dynamic_client.rb

```diff
@@ -36,6 +36,7 @@ def sdk_client(swagger, engine)
       conn.adapter :rack, engine
     end
 
+    LookerSDK.reset!
```

Comment:
I had intermittent failures because of test re-ordering during randomized runs - something to do with mock state being re-used :-( This change wipes out any mock re-usage